frontend/src/metabase/query_builder/components/DatasetEditor/DatasetEditor.jsx

```diff
@@ -393,10 +405,8 @@ function DatasetEditor(props) {
       return false;
     }
     const hasFieldWithoutDisplayName = fields.some(f => !f.display_name);
```

Comment:
nit: I know, it's not your changes, but it's pretty close :) I prefer positive statements rather than negative because of better readability. E.g. here it could be 

```
const allFieldsHaveDisplayName = !fields.some(f => !f.display_name);

return allFieldsHaveDisplayName && isDirty;
```

Reply:
100% agreed :+1: bf764dd9cc11de44e5f3efacc3cc310e3ede8fa4